src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -655,13 +653,13 @@ public Void visitInstanceOfExpr(JsoniqParser.InstanceOfExprContext ctx) {
     @Override
     public Void visitTreatExpr(JsoniqParser.TreatExprContext ctx) {
         Expression mainExpression;
-        FlworVarSequenceType sequenceType;
+        SequenceType sequenceType;
```

Comment:
Can we move these 2 variable declarations inside the if statement as they have no functionality outside that scope.